activemodel/test/cases/complex_password_test.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require "cases/helper"
+
+class ComplexPasswordTest < ActiveModel::TestCase
+  class ComplexPasswordUser
+    include ActiveModel::Model
+    include ActiveModel::ComplexPassword
+
+    attr_accessor :password
+
+    has_strong_password
+  end
```

Comment:
I added this class inside the test so as not to change the `models/user` class on the fly